### DISCUSSION TITLE
No information about `NotServedByPagesError` in GitHub pages documentation

### DISCUSSION THREAD

**Discussion by TokugawaTakeshi** [community/community#35168] [Other Feature Feedback, Questions, & Ideas]
Creating the GitHub page, I has been told:

> Both yamatodaiwa.com and its alternate name are improperly configured
> Domain does not resolve to the GitHub Pages server. For more information, see [documentation](https://docs.github.com/articles/setting-up-a-custom-domain-with-github-pages/) (NotServedByPagesError).

If try to search **NotServedByPagesError** in GitHub Pages official documentation, no search results will be:

![image](https://user-images.githubusercontent.com/41653501/194031312-44db039e-c23d-4415-b0db-7c4415d908a3.png)

The interesting thing is despite the error page is displaying on specified domain!

### About GitHub pages limitations

> The CNAME file must contain the domain name only. For example, www.example.com, blog.example.com, or example.com.

To be honest, I has not understood this limitation correctly. 
I have apex domain ` yamatodaiwa.com` and also `es-extensions.yamatodaiwa.com`, but I want to associate with each of them _separate_ GitHub repositories. As I got experimentally, it is possible.

### DNS records

The current DNS records are:

![image](https://user-images.githubusercontent.com/41653501/194001667-be057c34-6798-4d6e-b804-77219e21d277.png)

The **A** recordings has been set according the [instructions for apex domain](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain). The **CNAME** records has been set according [Configuring a subdomain](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain) instuctions.

### Screenshot

![image](https://user-images.githubusercontent.com/41653501/194031554-be02fee6-754b-4054-9c9b-a3298b91bdc1.png)

**Comment by yoannchaudet** [comment]
DNS propagation is not immediate. GitHub will not be able to tell you if your DNS configuration is good until it resolves from our side of the Internet.

If you are curious about the code that leaks in the error message, it comes from https://github.com/github/pages-health-check/blob/e1d29db4510709f7bd8095a4b534575cb90b0f14/lib/github-pages-health-check/errors/not_served_by_pages_error.rb#L6.

Checking your site now, I can tell your certificate has been obtained so I assume DNS eventually propagated.

**Reply by alexellis** [reply]
I have run into this today, with a newly created record. DNS on my network propagates very quickly, but I often find that GitHub (including when dealing with webhook URLS/domains) can take hours and hours to update.

**Reply by TokugawaTakeshi** [reply]
I have faced with this problem again once changed the domain. A few days have passed, so this time the cause is definitely not the delays of DNS propagation. Where I can get the help now?

I have created the single image with configuration of 2 sites (apex domain + subdomain) in GitHub, domain names provider and GitHub documentation. Currently both domains not directed to GitHub pages.

![image](https://github.com/community/community/assets/41653501/10e35689-783a-44c2-89c7-86e6798f594e)

**Reply by yoannchaudet** [reply]
@TokugawaTakeshi This is what I get when I resolve your domain:

<img width="666" alt="image" src="https://github.com/community/community/assets/14911070/b47ba7a6-b034-4418-b4bd-9c480055d883">

That's not Pages.

Your domain is still managed there:

<img width="645" alt="image" src="https://github.com/community/community/assets/14911070/9f7de627-4a9f-4654-9cf6-a0892ad251ca">

**Reply by TokugawaTakeshi** [reply]
@yoannchaudet 
Thank you for the investigations. 

Does it mean that something wrong on the side of my domain provider? 
If so, I'll contact with the provider, but if I say just; "My domain don't resolved on GitHub page" I'll be answered: "I don't have not hear about GitHub page. Please contact to GitHub customer support". What I need to show them to prove that the problem is on their side?

**Reply by yoannchaudet** [reply]
From my part of the world, I see the following when accessing your domain:

<img width="1431" alt="image" src="https://github.com/community/community/assets/14911070/d212b149-f172-40cb-9f16-827dbcd83d04">

So yeah I believe you have a configuration issue somewhere or a domain transfer that is slow to happen.

**Comment by piadehart-lab** [comment]
NotServedByPagesError

**Reply by piadehart-lab** [reply]
NotServedByPagesError
